source/plugin/bukkit-sponge-compare.rst

```diff
@@ -0,0 +1,680 @@
+event.player.PlayerResourcePackStatusEvent             event.entity.living.humanoid.player.ResourcePackStatusEvent
+event.player.PlayerRespawnEvent                        event.entity.living.humanoid.player.RespawnPlayerEvent
+event.player.PlayerShearEntityEvent
+event.player.PlayerStatisticIncrementEvent             event.statistic.ChangeStatisticEvent.TargetPlayer
+event.player.PlayerTeleportEvent                       event.entity.DisplaceEntityEvent.Teleport.TargetPlayer
+event.player.PlayerToggleFlightEvent
+event.player.PlayerToggleSneakEvent
+event.player.PlayerToggleSprintEvent
+event.player.PlayerUnleashEntityEvent
+event.player.PlayerUnregisterChannelEvent
+event.player.PlayerVelocityEvent
+event.server.MapInitializeEvent
+event.server.PluginDisableEvent
+event.server.PluginEnableEvent
+event.server.PluginEvent
+event.server.RemoteServerCommandEvent
+event.server.ServerCommandEvent
+event.server.ServerEvent
+event.server.ServerListPingEvent                       event.server.ClientPingServerEvent
+event.server.ServiceEvent
+event.server.ServiceRegisterEvent
+event.server.ServiceUnregisterEvent
+event.vehicle.VehicleBlockCollisionEvent
+event.vehicle.VehicleCollisionEvent
+event.vehicle.VehicleCreateEvent
+event.vehicle.VehicleDamageEvent
+event.vehicle.VehicleDestroyEvent
+event.vehicle.VehicleEnterEvent
+event.vehicle.VehicleEntityCollisionEvent
+event.vehicle.VehicleEvent
+event.vehicle.VehicleExitEvent
+event.vehicle.VehicleMoveEvent
+event.vehicle.VehicleUpdateEvent
+event.weather.LightningStrikeEvent
+event.weather.ThunderChangeEvent
+event.weather.WeatherChangeEvent                       event.world.ChangeWorldWeatherEvent
+event.weather.WeatherEvent
+event.world.ChunkEvent                                 event.world.chunk.TargetChunkEvent
+event.world.ChunkLoadEvent                             event.world.chunk.LoadChunkEvent
+event.world.ChunkPopulateEvent                         event.world.chunk.PopulateChunkEvent
+event.world.ChunkUnloadEvent                           event.world.chunk.UnloadChunkEvent
+event.world.PortalCreateEvent
+event.world.SpawnChangeEvent
+event.world.StructureGrowEvent
+event.world.WorldEvent                                 event.world.TargetWorldEvent
+event.world.WorldInitEvent                             event.world.ConstructWorldEvent
+event.world.WorldLoadEvent                             event.world.LoadWorldEvent
+event.world.WorldSaveEvent                             event.world.SaveWorldEvent
+event.world.WorldUnloadEvent                           event.world.UnloadWorldEvent
+event.Cancellable                                      event.Cancellable
+event.Event                                            event.Event
+event.EventException
+event.EventHandler                                     event.Listener
+event.EventPriority                                    event.Order
+event.HandlerList
+event.Listener
+generator.BlockPopulator
+generator.ChunkGenerator
```

Comment:
Replaced by `WorldGenerator`

This also has two sub-interfaces in bukkit which are:
`ChunkGenerator.BiomeGrid` -> `BiomeArea`
`ChunkGenerator.ChunkData` - > `BlockVolume`